packages/cache/src/index.js

```diff
@@ -56,11 +56,14 @@ let createCache = (options: Options): EmotionCache => {
   if (isBrowser && !movedStyles && key === 'css') {
     movedStyles = true
 
-    const ssrStyles = document.querySelectorAll(`style[data-emotion]`)
+    const ssrStyles = document.querySelectorAll(
+      `style[data-emotion]:not([data-s])`
+    )
     // get SSRed styles out of the way of React's hydration
     // document.head is a safe place to move them to
     Array.prototype.forEach.call(ssrStyles, (node: HTMLStyleElement) => {
       ;((document.head: any): HTMLHeadElement).appendChild(node)
+      node.dataset.s = 1
```

Comment:
Let's use setAttribute here instead of dataset so that this doesn't need to be polyfilled if people need to support ie10(iirc, the only things people would need to polyfill rn would be WeakMap and Map to be support ie10 and given there's basically no cost to using setAttribute, we should just do it IMO)